src/nxt_listen_socket.h

```diff
@@ -32,6 +32,10 @@ typedef struct {
 
     uint8_t                   socklen;
     uint8_t                   address_length;
+
+    nxt_socket_error_t        error;
+    u_char                    *start;
```

Comment:
@hongzhidao, the structure members `start` and `end` themselves have been moved here.